src/main/kotlin/mate/academy/model/impl/RightTriangle.kt

```diff
@@ -0,0 +1,21 @@
+package mate.academy.model.impl
+
+import mate.academy.model.Color
+import mate.academy.model.Figure
+
+private const val HALF = 2
```

Comment:
Moving this value to constant is overengineering a little because formula for rectangle area is not changing.
This is not mistake just FYI